docs/SRS-Volere/SRS.tex

```diff
@@ -190,15 +190,27 @@ \subsection{Adaptability Requirements}
 
 \section{Security Requirements}
 \subsection{Access Requirements}
-\lips
+\begin{itemize}
+    \item \textbf{ACR1}: The application shall require acceptance of terms and services to allow access to application's database at all times. 
+\end{itemize}
```

Comment:
Do you think we maybe need to include something like email and password somewhere for application access?

Reply:
thats the thing. I originally tried to include that, but didn't really know if we were going to make out app require login etc. For our purposes, I assumed we didn't need to, and we just need them to accept our terms and services.

Reply:
Looks like Allan included signing in as part of his sections and as a result, I also added them into my use case section. I think we need to have some sort of requirement where "Users will be required to sign in to access application features" or something like that.